.github/workflows/mvn-verify-check.yml

```diff
@@ -164,7 +164,8 @@ jobs:
           # verify Scala 2.13 build files
           ./build/make-scala-version-build-files.sh 2.13
           # verify git status
-          if ! git diff --exit-code 'scala2.13/*'; then
+          if [ -n "$(echo -n $(git status -s | grep 'scala2.13'))" ]; then
+              git add -N scala2.13/* && git diff 'scala2.13/*'
```

Comment:
nit: * is unnecessary  for git add 

As a more compact fix we could just insert  before L167 in the current code

`git add -N scala2.13`  

which will be a NOP if there are no untracked files without changing anything else

Changes to this file don't affect the internal CI  so adding [skip ci] to the PR title to save resources.